velox/functions/prestosql/window/tests/WindowTestBase.h

```diff
@@ -53,16 +53,20 @@ static std::vector<std::string> kSortOrderBasedOverClauses = {
 static std::vector<std::string> kFrameOverClauses = {
     "partition by c0 order by c1, c2",
     "partition by c1 order by c0, c2",
+    "partition by c0 order by c2, c1",
+    "partition by c1 order by c2, c0",
     // No partition by clause.
-    "order by c0 asc, c1 desc, c2",
-    "order by c1 asc, c0 desc, c2",
+    "order by c0 asc, c1 desc, c2 asc",
```

Comment:
Maybe add nulls first/last over some combinations of asc/desc.

Reply:
Done, thanks!